tests/cli/test_rasa_evaluate_markers.py

```diff
@@ -47,58 +47,62 @@ def write_markers_config_to_yaml(
 def test_evaluate_markers_help(run: Callable[..., RunResult]):
     output = run("evaluate", "markers", "--help")
 
-    help_text = """usage: rasa evaluate markers [-h] [-v] [-vv] [--quiet] [--config CONFIG]
-                             [--no-stats | --stats-file-prefix [STATS_FILE_PREFIX]]
-                             [--endpoints ENDPOINTS] [-d DOMAIN]
-                             output_filename {first_n,sample,all} ..."""
+    help_text = """usage: rasa evaluate markers [-h] [-v] [-vv] [--quiet]
+    {first_n,sample_n,all} ..."""
 
-    lines = help_text.split("\n")
+    lines = [line.strip() for line in help_text.split("\n")]
     # expected help text lines should appear somewhere in the output
-    printed_help = set(output.outlines)
+    printed_help = set([line.strip() for line in output.outlines])
     for line in lines:
         assert line in printed_help
 
 
 def test_evaluate_markers_first_n_help(run: Callable[..., RunResult]):
     # We need to specify an output_filename as that's the first positional parameter
-    output = run("evaluate", "markers", "--no-stats", "test.csv", "first_n", "--help")
+    output = run("evaluate", "markers", "first_n", "--help")
 
-    help_text = """usage: rasa evaluate markers output_filename first_n [-h] [-v] [-vv] [--quiet]
-                                                     count"""
+    help_text = """usage: rasa evaluate markers first_n [-h] [-v] [-vv] [--quiet]
+    [--config CONFIG]
+    [--no-stats | --stats-file-prefix [STATS_FILE_PREFIX]]
+    [--endpoints ENDPOINTS] [-d DOMAIN]
+    count output_filename"""
 
-    lines = help_text.split("\n")
+    lines = [line.strip() for line in help_text.split("\n")]
     # expected help text lines should appear somewhere in the output
-    printed_help = set(output.outlines)
+    printed_help = set([line.strip() for line in output.outlines])
     for line in lines:
         assert line in printed_help
 
 
-def test_evaluate_markers_sample_help(run: Callable[..., RunResult]):
+def test_evaluate_markers_sample_n_help(run: Callable[..., RunResult]):
     # We need to specify an output_filename as that's the first positional parameter
-    output = run("evaluate", "markers", "test.csv", "sample", "--help")
+    output = run("evaluate", "markers", "sample_n", "--help")
 
-    help_text = """usage: rasa evaluate markers output_filename sample [-h] [-v] [-vv] [--quiet]
-                                                    [--seed SEED]
-                                                    count"""
+    help_text = """usage: rasa evaluate markers sample_n [-h] [-v] [-vv] [--quiet] [--seed SEED]
```

Comment:
Not entirely sure why these have become un-indented but tests seem to pass?

Reply:
They don't have to be un-indented. I just had to change the content and ended up formatting differently. I am stripping leading and trailing whitepace in the test so that the indentation wouldn't matter, just the content.